lib/l10n_utils/management/commands/fluent.py

```diff
@@ -0,0 +1,83 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Convert a template to use Fluent for l10n'
+    requires_system_checks = False
+
+    def add_arguments(self, parser):
+        subparsers = parser.add_subparsers(
+            title='subcommand', dest='subcommand'
+        )
+        subparsers.add_parser('help')
+
+        recipe_parser = subparsers.add_parser(
+            'recipe',
+            description='Create migration recipe from template'
+        )
+        recipe_parser.add_argument('template', type=Path)
+
+        ftl_parser = subparsers.add_parser(
+            'ftl',
+            description='Create Fluent file with existing recipe'
+        )
+        ftl_parser.add_argument(
+            'recipe_or_template', type=Path,
+            help='Path to the recipe or the template from which the recipe was generated'
+        )
+        ftl_parser.add_argument(
+            'locales', nargs='*', default=['en'], metavar='ab-CD',
+            help='Locale codes to create ftl files for'
+        )
+
+        template_parser = subparsers.add_parser(
+            'template',
+            description='Create template_ftl.html file with existing recipe'
+        )
+        template_parser.add_argument('template', type=Path)
+
+    def handle(self, subcommand, **kwargs):
+        if subcommand in (None, 'help'):
+            return self.handle_help(**kwargs)
+        if subcommand == 'recipe':
+            return self.create_recipe(**kwargs)
+        if subcommand == 'ftl':
+            return self.create_ftl(**kwargs)
+        if subcommand == 'template':
+            return self.create_template(**kwargs)
+
+    def handle_help(self, **kwargs):
+        self.stdout.write('''\
```

Comment:
I find it best to use `textwrap.dedent` in these situations so that you can maintain the file indentation.